include/nn/uds/Cafe.h

```diff
@@ -0,0 +1,471 @@
+#pragma once
+
+#include <wut.h>
+#include <nn/result.h>
+#include <nn/uds/NodeInformation.h>
+#include <nn/cfg/CTR.h>
+
```

Comment:
You should add something like this here, so this gets shown on doxygens modules overview:
```
/**
 * \defgroup nn_uds
 * \ingroup nn_uds
 * UDS Network functions (see nn::uds)
 * @{
 */
 ```
and
```
/** @} */
```
at the end of the file.